<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora IMC</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(33,33,33,1);
color:  rgb(170, 170, 255);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>321</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="4">
     <widget class="QLabel" name="textoIMC_magreza">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string>&lt; 18.5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="textoP_IDEAL_magreza">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="textoPESO">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(200, 200, 255);
color: rgba(33,33,33,1);
border-top-right-radius: 10px;
</string>
      </property>
      <property name="text">
       <string>PESO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="textoIMC_obesidade">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string>&gt; 30</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="textoP_IDEAL_obesidade">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="textoIMC_sobrepeso">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string>24.9 a 30</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="vazia">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(200, 200, 255);
border-top-left-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="textoP_IDEAL_normal">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="textoMagreza">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string>Magreza</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="textoP_IDEAL_sobrepeso">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="textoObesidade">
      <property name="maximumSize">
       <size>
        <width>86</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string>Obesidade</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="textoIMC_normal">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string>18.5 a 24.9</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="textoSobrepeso">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string>Sobrepeso</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="textoNormal">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid rgb(200, 200, 255);</string>
      </property>
      <property name="text">
       <string>Normal</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="textoIMC">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(200, 200, 255);
color: rgba(33,33,33,1);
</string>
      </property>
      <property name="text">
       <string>IMC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="inicioDescricao">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top: 2px solid rgb(200,200,255);</string>
   </property>
   <property name="text">
    <string>&amp;nbsp;De acordo com a Organização Mundial da Saúde, seu IMC está abaixo do recomendado para a sua altura. Para atingir um valor de IMC normal, seu peso deve estar entre 81.6 e 109.8 kg.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="inicioRelatorio">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>45</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Olá, Henrique Fernades da Silva Alves.

Seu resultado foi 25.5 kg/m²</string>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
